src/content/platform-integration/android/call-jetpack-apis.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Calling JetPack APIs"
+description: "Utilize the latest Android APIs from your Dart code"
+---
+
+<?code-excerpt path-base="platform_integration"?>
+
+Flutter apps running on Android can always make use of the
+latest APIs, the first day they are released on Android, no
+matter what. This page outlines available ways to invoke
+Android-specific APIs.
+
+## Use an existing solution
+
+In most scenarios, you can [use a plugin][use-a-plugin] to
+invoke native APIs without writing any custom boilerplate or
+glue code yourself.
+
+[use-a-plugin]: /platform-integration/android/call-jetpack-apis#use-a-plugin
+
+### Use a plugin
+
+Using a plugin is often the easiest way to access native
+APIs, regardless of where your Flutter app is running. To
+use plugins, visit [pub.dev][pub] and search for
+the topic you need. Most native features, including accessing
+common hardware like GPS, the camera, or step counters are
+supported by robust plugins.
+
+For complete guidance on adding plugins to your Flutter app,
+see the [Using packages documentation][packages].
+
+[packages]: /packages-and-plugins/using-packages
+[pub]: {{site.pub}}
+
+Not all native featurers are supported by plugins, especially
+new features immediately after release. In any scenario where
+your desired native feature is not covered by a package on
+[pub.dev][pub], continue on to the following sections.
+
+## Creating a custom solution
+
+Not all scenarios and APIs will be supported by
+existing solutions; but luckily, you can always add whatever
+support you need. The ensuing sections describe two different
+ways to call native code from Dart.
+
+:::note
+Neither solution below is inherently better or worse than 
+existing plugins, because all plugins use one of the ensuing
+options.
+:::
+
+### Call native code directly via FFI
+
+The most direct and efficient way to invoke native APIs is by
+calling it dirrectly, via FFI. This links your Dart executable
+to any specified native code at compile-time, allowing you to 
+call it directly from the UI thread through a modicum of glue
```

Comment:
Many folks that read these docs aren't native English speakers. Consider limiting our vocabulary to stay accessible. For example, instead of "through a modicum of" consider "through a little".

I know this hampers the personality and voice of our docs :(

Reply:
No, that's a good call. I'll remove "modicum".